packages/2019-template/src/components/TemplateAPICard/index.js

```diff
@@ -11,13 +11,7 @@ import api from "../../state/template-api-data/api";
 const TemplateAPICard = ({ init, data, Layout }) => {
   useEffect(() => {
     init();
-  }, [
-    /*
-    https://reactjs.org/docs/hooks-effect.html#tip-optimizing-performance-by-skipping-effects
-
-    Add second argument to prevent useEffect running init() again
-    */
-  ]);
+  }, [init]);
```

Comment:
This line needs to be reverted. You may will probably need to add an eslint-disable-next-line. Written like this, there are cases where init can be run again and we don't want that behavior